AcloConcurrency/AcloConcurrency/Collection/Cocktails/CocktailDescription/ViewModel/CocktailDescriptionViewModel.swift

```diff
@@ -0,0 +1,3 @@
+import Combine
+
+public class CocktailDescriptionViewModel: ObservableObject {}
```

Comment:
Where we use this file?

Reply:
we will use this file in the future, when the we will init and fetch description